dolfyn/tools/psd.py

```diff
@@ -1,25 +1,9 @@
 import numpy as np
```

Comment:
The functions here that have been moved to internal (leading "_" added to function name) should all probably be included in the API. They are useful, I think.